clover-conversion/Clover-config.md

```diff
@@ -488,10 +491,11 @@ Note: Finding CPUID's for Intel can be a bit harder than looking at Intel ARK, e
 
 **BooterConfig**:
 
-* `NVRAM -> Add -> 4D1EDE05-38C7-4A6A-9CC6-4BCCA8B38C14-> UIScale`:
-
-  * 0x28: `Data | <01>`
-  * 0x2A: `Data | <02>`
+* `UEFI -> Quirks -> UIScale | Integer |`
```

Comment:
`UIScale` got dropped from `NVRAM -> Add -> 4D1EDE05-38C7-4A6A-9CC6-4BCCA8B38C14-> UIScale`. That's what also `ocvalidate` tells as well as [Sample.plist](https://github.com/acidanthera/OpenCorePkg/blob/master/Docs/Sample.plist#L1646=)